components/TransLine.tsx

```diff
@@ -0,0 +1,21 @@
+import { Trans } from 'react-i18next'
+import React from 'react'
+
+export interface TransLineProps {
+  i18nString: string // Expects an i18n string
```

Comment:
Nitpick: what do you think about calling this `i18nKey` instead? At first blush I don't think I know which side of the object `string` refers to.